modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,159 @@
+= Use Studio to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To use Anypoint Studio to configure a connector:
+
+. Install the connector.
+. Configure an input source for the connector.
+. Configure the connector.
+
+Follow these best practices when you configure an input source and connector in Studio:
+
+*  Configure an associated global element for the input source and connector.
+* Create a `.yaml` file or `.properties` file to contain your properties, and then reference the file from the configuration.
+* Configure a reconnection strategy for the input source and target connector.
+
+For information about these best practices, see <<introduction/intro-connector-configuration-overview.adoc#best-practices,Best Practices for Configuring Connectors>>.
+
+== Install the Connector
+
+Installing the connector adds it to a Studio project. Studio provides two ways to install a connector:
```

Comment:
There's one more, and actually the one that I would recommend users to use: The **Manage Mule Project** option: 
![manage-mule-project](https://user-images.githubusercontent.com/16763154/72365808-b6257c80-36d7-11ea-9451-d384720ea590.png)

This option lets users configure all the dependencies of their project:

![modules-management](https://user-images.githubusercontent.com/16763154/72366305-c2f6a000-36d8-11ea-9869-f1be35572191.png)

In the **Modules** option users can:
1. Add or remove modules/connectors.
2. Look in Exchange for newer versions of the connectors and update them from there.

Reply:
Leaving out for now because of the issue with the connector not showing up in the palette.